webapi/tct-colors-css3-tests/css3-color-app/testscripts/t31-color-currentColor-b.feature

```diff
@@ -1,7 +1,6 @@
-Feature: w3c-webstorage
- Scenario: storage insurance on trip
+Feature: css3-color
+ Scenario: t31 color currentColor b
    When launch "css3-color-app"
      And I go to "csswg/t31-color-currentColor-b.htm"
      And I save the page to "css3_colors_tests_entry1"
```

Comment:
To make the baseline data consistent with case name, please update the caseID to htm file name in all xxx.feature file, and update baseline data name